test/intl402/Locale/extensions-grandfathered.js

```diff
@@ -31,6 +31,8 @@ info: |
     ...
 
 features: [Intl.Locale]
+locale:
```

Comment:
Intl.Locale supports all locales, regardless of what's supported on a given system. I think this frontmatter should only be used on tests which are *not* required to work, e.g., in your PluralRules example above.